draft-packages/split-button/KaizenDraft/SplitButton/styles.scss

```
@@ -101,32 +112,37 @@ $side-padding: 3/4 * $ca-grid;
 
 .buttonDefault {
   @include button-mixing(
-    $background: $kz-color-white,
-    $border-color: $kz-color-wisteria-200,
-    $color: $kz-color-wisteria-800,
-    $hover-background: $kz-color-stone,
-    $hover-border-color: $kz-color-wisteria-400,
-    $active-background: $kz-color-ash,
-    $active-border-color: $kz-color-wisteria-500,
-    $disabled-background: $kz-color-white,
-    $disabled-border-color: $kz-color-wisteria-200,
-    $disabled-color: $kz-color-wisteria-300
+    $background: $kz-var-color-white,
```

Adding one of these to make cleanup easier:

```suggestion
    // Use of variable fallbacks to switch between Heart and Zen themes (use Zen as default fallback)
    $background: $kz-var-color-white,
```